tests/test_scripts.py

```diff
@@ -6,24 +6,50 @@
 """
 
 import os.path as osp
+import sys
 import tempfile
 from collections import Counter
 from typing import List, Optional
+from unittest import mock
 
 import pandas as pd
 import pytest
 import ray.tune as tune
+import sacred
 
-from imitation.scripts.analyze import analysis_ex
-from imitation.scripts.eval_policy import eval_policy_ex
-from imitation.scripts.expert_demos import expert_demos_ex
-from imitation.scripts.parallel import parallel_ex
-from imitation.scripts.train_adversarial import train_ex
+from imitation.scripts import (
+    analyze,
+    eval_policy,
+    expert_demos,
+    parallel,
+    train_adversarial,
+)
+
+ALL_SCRIPTS_MODS = [analyze, eval_policy, expert_demos, parallel, train_adversarial]
+
+
+@pytest.fixture(autouse=True)
+def sacred_capture_use_sys():
```

Comment:
Should probably also have a TODO/SOMEDAY on it since a workaround for a bug in Sacred.